content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,225 @@
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Integrations]
+
+Docs: provide a link to the updated docs	
+##### 🏗️ High-leverage ways to improve your entire codebase
+
+### Make changes across all of your codebases at enterprise scale with server-side Batch Changes (beta)
+Batch Changes can now run across thousands of repositories with a better development experience. Previously, you had to run src-cli locally to create batch changes. That could take an impractical amount of time or be brittle for large or resource-intensive batch changes. By running Batch Changes server-side you can:
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience with the streaming of logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs) as well. One or more executors are available by default on Cloud instances of Sourcegraph.
+
+[Batch Changes]
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+- New packaging updates
+
+##### ☁️ Enterprise Cloud dedicated(/managed?) instances
+
+- [Secure, easy, and scalable Sourcegraph Cloud dedicated(/managed?) instances for the enterprise](/blog/enterprise-cloud)
+
+Headline: [Do X] with [Z]
+Description: 2 - 4 sentences on the feature, functionality and benefits
+Media: provide a screenshot, gif, or video to help support the copy
+Tag by product: [Code Search], [Batch Changes], [Code Navigation], [Code Insights], [Admin]
+Docs: provide a link to the updated docs
+
+##### 💖 For admins
+
+### Get a deep understanding of usage and measure the value of our platform with new and enhanced in-product analytics
+With 3.42, we introduced an all-new analytics including [XYZ]. Our new analytics make it easier for admins to understand user engagement, measure efficiency in terms of time saved, and help calculate the value of utilizing Sourcegraph and its features. We’ve significantly improved our in-product analytics to include the projected value and dev-time-saved for all Sourcegraph features, the ability to export data, individual user usage table, and much more. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Admin]
+
+Docs: provide a link to the updated docs
```

Comment:
@ryphil to be completed with the new analytics that were added in 3.42, media and a link to the updated docs